src/main/java/com/example/gradecalculator/entities/Subject.java

```diff
@@ -3,38 +3,52 @@
 import jakarta.persistence.*;
 
 @Entity
-@Table(name = "subject")
+@Table( name = "subject")
 public class Subject {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @Column(name = "id")
     private Long id;
 
-    @Column(name = "name")
+    @Column(unique = true)
     private String name;
 
-    public Subject() {
-    }
+    private String description;
+
+    //private int creditValue;
```

Comment:
please remove commented out code as it is not used

Reply:
I wanted first ask if we need this parameter but I found out just know that we need it for calculation of GPA. Thus I will remove commenting.